backend/src/entities/mail.py

```diff
@@ -0,0 +1,19 @@
+from flask_mail import Mail, Message
+from main import mail
+
+
+# app.config['MAIL_SERVER']='smtp.gmail.com'
+# app.config['MAIL_PORT'] = 465
+# app.config['MAIL_USERNAME'] = 'indium@gmail.com'
+# app.config['MAIL_PASSWORD'] = 'xxxxxx'
+# app.config['MAIL_USE_TLS'] = False
+# app.config['MAIL_USE_SSL'] = True
+#
+# mail= Mail(app)
+
+def send_mail():
+   msg = Message('test', sender = 'indium@gmail.com', recipients = ['xxx@indiumsoft.com'])
```

Comment:
the subject, from, to and any other information should come from the function call.

Reply:
updated. please check